otter_welcome_buddy/common/constants.py

```diff
@@ -8,3 +13,9 @@ class CronExpressions(Enum):
 
 
 COMMAND_PREFIX: str = "!"
+
+WELCOME_MESSAGES = os.environ.get("WELCOME_MESSAGES", "").split(",")
```

Comment:
Can you add where should I pull this environment variables?

Reply:
Wdym?

Reply:
Where are we going to store this information so when I would like to run the project I can pull the WELCOME_MESSAGES, keyvault?

Reply:
For sure, DM me and I will put the secrets there and grant you access.